### DISCUSSION TITLE
Regarding OpenTCS APIs

### DISCUSSION THREAD

**Discussion by sirisatwika** [openTCS/opentcs#54] [Q&A]
1. How do we connect OpenTCS with Simulators like Unity 3D or FlexSIM ?
2.  How to access the Web API to get the vehicle data and store them in a database ?

**Comment by swltr** [comment]
1. See the [section in the Developer's Guide](https://www.opentcs.org/docs/5.6/developer/developers-guide/opentcs-developers-guide.html#section-kernel-api) and the [web API specification](https://www.opentcs.org/docs/5.6/developer/service-web-api-v1/index.html?docExpansion=list).
2. It is unclear what your question is about.

**Reply by sirisatwika** [reply]
2. How to get the vehicle specific data from OpenTCS and store that data in a database. Can you provide with any Connectors.

**Reply by sirisatwika** [reply]
Thankyou for the response....

**Reply by swltr** [reply]
As mentioned above: see the [web API specification](https://www.opentcs.org/docs/5.6/developer/service-web-api-v1/index.html?docExpansion=list) for what data you can retrieve via the web API. How you process that data, i.e. by putting it into a database, is out of scope of the openTCS project.
